### PR TITLE
remove.php/download.php examples

### PR DESCRIPTION
**Pull request by dimaninc**

### CODE REVIEW

File: sample-project/remove.php [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+use Uploadcare\Api;
+
+// create object instance for Api.
+$api = new Api(UC_PUBLIC_KEY, UC_SECRET_KEY);
+
+// get all files iterator
+$files = $api->getFileList();
+
+// get all files iterator
+$files = $api->getFileList();
```
Comment (dmitry-mukhin) [inline sample-project/remove.php]: :)

File: sample-project/remove.php [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+// requesting autoloader that got uploadcare in there
+require_once 'vendor/autoload.php';
+// using api
+use Uploadcare\Api;
+
+// create object instance for Api.
+$api = new Api(UC_PUBLIC_KEY, UC_SECRET_KEY);
+
+// get all files iterator
+$files = $api->getFileList();
```
Comment (dmitry-mukhin) [inline sample-project/remove.php]: please add filter example, e.g. file uploaded earlier than X days ago

File: sample-project/remove.php (hunk 1 above)
Comment (dimaninc) [inline sample-project/remove.php]: oops :)

File: sample-project/remove.php (hunk 2 above)
Comment (dimaninc) [inline sample-project/remove.php]: done

File: sample-project/remove.php [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+// using api
+use Uploadcare\Api;
+
+// create object instance for Api.
+$api = new Api(UC_PUBLIC_KEY, UC_SECRET_KEY);
+
+// get all files iterator
+// using filter: only files uploaded earlier than 14 days ago
+$files = $api->getFileList(array(
+  'to' => new \DateTime('-14 days')
```
Comment (dmitry-mukhin) [inline sample-project/remove.php]: O_o

**Pull request merged**